### PR TITLE
converted couchbase-3.2 test from groovy to java

### PR DESCRIPTION
**Pull request by quinoant**
related to #7195

### CODE REVIEW

File: instrumentation/couchbase/couchbase-3.2/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/couchbase/v3_2/CouchbaseClient32Test.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.couchbase.v3_2;
+
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
```
Comment (mateuszrzeszutek) [inline instrumentation/couchbase/couchbase-3.2/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/couchbase/v3_2/CouchbaseClient32Test.java:11]: There already is a copyright header on the top of the file
```suggestion
```

**Review by mateuszrzeszutek — APPROVED** [review]

**Comment by mateuszrzeszutek** [comment]
Thanks @quinoant !

File: instrumentation/couchbase/couchbase-3.2/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/couchbase/v3_2/CouchbaseClient32Test.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  private static final InstrumentationExtension testing = AgentInstrumentationExtension.create();
+
+  private static final Logger logger = LoggerFactory.getLogger("couchbase-container");
+
+  static CouchbaseContainer couchbase;
+  static Cluster cluster;
+  static Collection collection;
+
+  @BeforeAll
+  static void setupSpec() {
```
Comment (laurit) [inline instrumentation/couchbase/couchbase-3.2/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/couchbase/v3_2/CouchbaseClient32Test.java:45]: `setupSpec` and `cleanupSpec` are spock conventions, for java tests we use `setUp` or `setup` and `cleanUp`, `cleanup` or `tearDown`

File: instrumentation/couchbase/couchbase-3.2/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/couchbase/v3_2/CouchbaseClient32Test.java (hunk 1 above)
Comment (quinoant) [reply instrumentation/couchbase/couchbase-3.2/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/couchbase/v3_2/CouchbaseClient32Test.java]: Thanks for the tip, slipped right by me

File: instrumentation/couchbase/couchbase-3.2/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/couchbase/v3_2/CouchbaseClient32Test.java (hunk 2 above)
Comment (quinoant) [reply instrumentation/couchbase/couchbase-3.2/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/couchbase/v3_2/CouchbaseClient32Test.java]: Thanks for letting me know about the conventions, very helpful

**Review by mateuszrzeszutek — APPROVED** [review]

**Pull request merged**